themes/digital.gov/layouts/guides/list.html

```diff
@@ -7,31 +7,25 @@
 
 
   <main role="main" id="main-content">
-    <article>
-      <div class="grid-container grid-container-desktop">
-        {{- partial "partials/core/usa-breadcrumbs.html" . -}}
-        <div class="grid-row grid-gap-4">
-          <div class="grid-col-12 tablet-lg:grid-col-2">
-            {{- partial "core/guides/guide-nav.html" . -}}
-          </div>
+    <article class="grid-container grid-container-desktop">
+      {{- partial "partials/core/usa-breadcrumbs.html" . -}}
+      <section class="grid-row grid-gap-4">
+        {{- partial "core/guides/guide-nav.html" . -}}
```

Comment:
It might be better to move the nav's layout grid classes to a wrapper here.

Reply:
Gotcha, I've added a wrapper `div` with the layout grid classes.